### PR TITLE
Update README.md to include explanatory function names per #200

### PR DESCRIPTION
**Pull request by rouzbeh84**
updating function name per #200 thanks to @marzelin to clean up some issues

### CODE REVIEW

**Comment by ryanmcdermott** [comment]
Thanks! My only suggestion is it might be just a tad clearer to have `isActive` still be `isActiveClient`. We may have another method down the line which will be checking if an employee is active, as an example. What are your thoughts?

**Comment by rouzbeh84** [comment]
I personally agree, I was sticking to the issue discussion suggestions but this does make more sense. While typing it up I thought of `isActiveClientActive || isClientEmailActive' but was hesitant in getting *too* descriptive in making it so I think this is a nice happy medium :)

File: README.md [hunk 1]
```diff
@@ -273,13 +273,13 @@ function emailClients(clients) {
 
 **Good:**
 ```javascript
-function emailClients(clients) {
+function emailActiveClients(clients) {
   clients
-    .filter(isClientActive)
+    .filter(isActive)
```
Comment (ryanmcdermott) [inline README.md]: Can you change this to `isActiveClient`? Thanks!

Comment (lavinle) [inline README.md]: trying to stay active.... i amnot sure about everything here.. need my time

File: README.md [hunk 2]
```diff
@@ -275,7 +275,7 @@ function emailClients(clients) {
 ```javascript
 function emailActiveClients(clients) {
   clients
-    .filter(isActive)
+    .filter(isActiveClient)
```
Comment (rouzbeh84) [inline README.md]: Should be all set 👍

**Comment by rouzbeh84** [comment]
Wait, did I miss something on this? The names are so similar it feels like I missed something haha

**Comment by ryanmcdermott** [comment]
Sorry for the delay! Thank you @rouzbeh84!

**Comment by rouzbeh84** [comment]
No worries at all! I just kept seeing it and was hoping I didn't mess it up somehow haha. Great project! keep up the awesome work!

**Pull request merged**